applications/nrf_desktop/src/modules/ble_discovery.c

```diff
@@ -0,0 +1,291 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <misc/byteorder.h>
+#include <zephyr/types.h>
+
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/gatt_dm.h>
+
+#define MODULE ble_discovery
+#include "module_state_event.h"
+
+#include "ble_event.h"
+#include "ble_discovery_def.h"
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_BLE_DISCOVERY_LOG_LEVEL);
+
+static u16_t peer_vid;
+static u16_t peer_pid;
+static struct bt_conn *discovering_peer_conn;
+
+static struct k_work hids_discovery_start;
+
+#define POS sizeof(u8_t)
```

Comment:
`POS` says very little about what it actually is :)
sorry, my suggestion in earlier comment was not very elaborate...

Reply:
It was used only one line below, it's why I decided to use short define.
I changed it to be more informative.